diff -up gnome-session-3.28.1/tools/gnome-session-check-accelerated-gles-helper.c.fix-crash-on-no-display gnome-session-3.28.1/tools/gnome-session-check-accelerated-gles-helper.c
--- gnome-session-3.28.1/tools/gnome-session-check-accelerated-gles-helper.c.fix-crash-on-no-display	2018-10-02 10:29:54.757972502 -0400
+++ gnome-session-3.28.1/tools/gnome-session-check-accelerated-gles-helper.c	2018-10-02 10:30:18.859714829 -0400
@@ -178,7 +178,7 @@ main (int argc,
         g_option_context_add_main_entries (context, entries, NULL);
 
         if (!g_option_context_parse (context, &argc, &argv, &error)) {
-                g_error ("Can't parse command line: %s\n", error->message);
+                g_debug ("Can't parse command line: %s\n", error->message);
                 g_error_free (error);
                 goto out;
         }
